packages/grafana-ui/src/components/Forms/Field.tsx

```diff
@@ -30,6 +32,8 @@ export interface FieldProps extends HTMLAttributes<HTMLDivElement> {
   horizontal?: boolean;
   /** make validation message overflow horizontally. Prevents pushing out adjacent inline components */
   validationMessageHorizontalOverflow?: boolean;
+  /** Adds tooltip next to the label */
+  tooltipMessage?: string;
```

Comment:
For the horizontal form components we use `description` prop to provide extra context, maybe you could use it instead of adding the tooltip? I don't think we use tooltips in horizontal form components and only use them in the inline form components in the design system, but maybe @JoaoSilvaGrafana can confirm.

Reply:
Yes preferably we would not add a tooltip to `Field` without a reasonable check on how this would work fully. It would need to be documented on when you use tooltip vs description and what does this mean for all the other places that use description, if some of them would be changed to tooltip or not. So if we could use description here it would be best